specs/bellatrix/beacon-chain.md

```diff
@@ -320,6 +337,30 @@ def notify_new_payload(self: ExecutionEngine, execution_payload: ExecutionPayloa
     ...
 ```
 
+#### `is_valid_block_hash`
+
+```python
+def is_valid_block_hash(self: ExecutionEngine, execution_payload: ExecutionPayload) -> bool:
+    """
+    Return ``True`` if and only if ``execution_payload.block_hash`` is computed correctly.
+    """
+    ...
+```
+
+#### `verify_and_notify_new_payload`
+
+```python
+def verify_and_notify_new_payload(self: ExecutionEngine,
+                                  new_payload_request: NewPayloadRequest) -> bool:
+    """
+    Return ``True`` if and only if ``new_payload_request`` is valid with respect to ``self.execution_state``.
+    """
+    assert self.is_valid_block_hash(new_payload_request.execution_payload)
+    assert self.notify_new_payload(new_payload_request.execution_payload)
+    ...
```

Comment:
What does the ... imply here?

Reply:
I wanted to imply that this is a pseudo-code-like function. 😅